envs/monkey_zoo/blackbox/island_client/monkey_island_client.py

```diff
@@ -85,3 +86,40 @@ def get_test_query_results(response):
     def is_all_monkeys_dead(self):
         query = {'dead': False}
         return len(self.find_monkeys_in_db(query)) == 0
+
+    def clear_caches(self):
+        """
+        Tries to clear caches.
+        :raises: If error (by error code), raises the error
+        :return: The response
+        """
+        response = self.requests.get("api/test/clear_caches")
+        response.raise_for_status()
+        return response
+
+    def time_all_report_pages(self):
```

Comment:
This whole method is specific to the performance test. You should move it there.

Reply:
OR even a better way would be to add `send_timed_get()` to the `monkey_island_requests.py`. Then use it from `performance_test.py`, pass URLs and so on.

Reply:
Done

Reply:
What's done?

Reply:
347941c7